content/it/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Comunicazione Piano di controllo - Nodo
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+
+Questo documento cataloga i percorsi di comunicazione tra il piano di controllo (_control plane_), in realtà l'apiserver, e il cluster Kubernetes. L'intento è di consentire agli utenti di personalizzare la loro installazione per rafforzare la configurazione di rete affinché il cluster possa essere eseguito su una rete non affidabile (o su IP completamente pubblici resi disponibili da un fornitore di servizi cloud).
+
+
+<!-- body -->
+
+## Dal Nodo al piano di controllo{#dal-nodo-al-piano-di-controllo}
```

Comment:
```suggestion
## Dal Nodo al control-plane{#dal-nodo-al-piano-di-controllo}
```
Once an english term is defined, IMO we should use it to make the document less awkward for users acquainted to reading Kubernetes docs/articles

Reply:
I agree and I change it accordingly but I understood from my previous translation that we should translate them all to the local language adding in parenthesis the en/standard word/expression only at the first occurrence

Reply:
My understanding that once an english term is defined we can use that without repeating the translitions

Reply:
I'm wondering what a Spanish speaker gets from having “control-plane” in the text rather than “piano di controllo”, which seems to me like the right term to use throughout.

If the original were in Spanish I wouldn't expect to see piano-di-controllo repeated through the English version of the document.

Reply:
@sftim I got several feedbacks from users telling us that exceeding in translating mainstream terms in Italian only creates confusion and makes the documentation look weird. 

It is a fact that most of the documentation is in English, most of the blogs are in English and Italian users working in IT are using a jargon of Italian and English mainstream terms (and this happens not only in Kubernetes).

Se, I think that as a contributor my goal is to meet the need of our users base finding the right balance between literal translation - which makes docs look weird - and the jargon IT users are used to. 

With regards to the hypothetical discussion about what if the original term was "piano di controllo", I would say that in real life this is not the case. At the end also Ensglish native speaker uses italian terms like pizza or pasta without translating them.

Reply:
Based on https://github.com/kubernetes/website/pull/22303#discussion_r453828496, does it make sense to use “control plane” in the page title also and then explain that term as an aside in the page introduction?

Reply:
ok I'll change the title as well using "control-plane". I'll use "piano di controllo" (control-plane) within the page at the occurrence and then always "control-plane". Deal?

Reply:
(sounds good to me - happy for Italian localization approvers to have a final say)

Reply:
done